dap_aria_mapping/analysis/app/pages/2_Change_Makers.py

```diff
@@ -18,6 +24,121 @@
     page_icon=icon
 )
 
+@st.cache_data(show_spinner = "Loading Data")
```

Comment:
I'm not sure what the effect is on performance, but since the output for these is a nx.Graph object, we should use st.cache_resource, as the graph object is not easily serializable and deserializable. Recommendation:
`@st.cache_resource(allow_output_mutation=True)`

Reply:
will change in 73